samtranslator/model/apigatewayv2.py

```diff
@@ -115,6 +117,13 @@ def __init__(  # type: ignore[no-untyped-def] # noqa: too-many-arguments
         if authorizer_type == "REQUEST":
             self._validate_lambda_authorizer()
 
+        if disable_function_default_permissions is not None:
+            sam_expect(
+                disable_function_default_permissions,
+                api_logical_id,
+                f"Authorizers.{name}.DisableFunctionDefaultPermissions",
+            ).to_be_a_bool()
```

Comment:
Would be clearer to add a `to_be_a_bool_or_none`.

Reply:
need to refactor sam_expect a bit more so going to make this a separate PR